pkg/pipelines/environment_test.go

```diff
@@ -0,0 +1,48 @@
+package pipelines
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEnv(t *testing.T) {
+
+	gitopsPath, cleanUp := fakeGitopsDir(t)
+	defer cleanUp()
+
+	envParameters := EnvParameters{
+		EnvName:    "dev",
+		GitOpsRepo: "org/gitops",
+		Output:     filepath.Dir(gitopsPath),
+	}
+	if err := Env(&envParameters); err != nil {
+		t.Fatalf("Env() failed :%s", err.Error())
```

Comment:
It feels odd to use testify's `assert.FileExists` below, but not to use https://godoc.org/github.com/stretchr/testify/assert#Nil here ?

At a minimum, you can remove `err.Error()` as the %s of an err will be the result of calling `.Error()` on it

(if you want to  understand why, https://github.com/golang/go/blob/master/src/fmt/print.go#L624)

Reply:
made the required changes. Removed err.Error() as the %s of an err